src/pages/profile.tsx

```diff
@@ -21,26 +20,29 @@ import UserIcon from "@/icons/UserIcon";
 import FemaleIcon from "@/icons/FemaleIcon";
 import MaleIcon from "@/icons/MaleIcon";
 import moment from "moment";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
 
 export default function Profile(props: { user: GetUserInfoResponse }) {
   const inputFileRef = useRef<any>();
-  const inputDateRef = useRef<any>();
 
   const [gender, setGender] = useState<Gender>(Gender.FEMALE);
   const [userImgId, setUserImgId] = useState<number>(-1);
   const [userImgUrl, setUserImgUrl] = useState<string>("");
+  const [birthDate, setBirthDate] = useState<Date | null>(null);
 
   useEffect(() => {
     const token = localStorage.getItem("token") || "";
     getUser(token)
       .then((user: any) => {
         if (user) {
+          setBirthDate(user.birthdate ? new Date(user.birthdate) : null);
           setUserImgId(user?.profileImage?.id);
           setGender(user?.gender);
           setUserImgUrl(user?.profileImage?.url);
         }
       })
-      .catch(() => {});
+      .catch(() => { });
```

Comment:
catch and ..

Reply:
ลบแน้วค้าบ

Reply:
LGTM kubbbbbbb

Reply:
อยากได้ลิง dev จัง

Reply:
https://dev.lifecycleapp.online/

Reply:
สงสัย feature ที่มันขอ access อัดเสียงเวลานอนอะ มันมี limitation อะไรป่ะ
ตอนนี้พอกดมันจะกดนับเวลานอนและก็อัดเสียงให้เลย ถ้ากดออก กะล็อคหน้าจอ
เข้าใจว่ามันยัง handle อัดเสียงต่อให้อยู่ (`Safari ip12 iOS 16.4.1`) มันขึ้นว่าใช้ไมค์อยู่

Reply:
ขก พิมอะโทรได้ป่าว